Content.Shared/SS220/MartialArts/MartialArtistComponent.cs

```diff
@@ -0,0 +1,36 @@
+// © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.SS220.MartialArts;
+
+[RegisterComponent, NetworkedComponent, Access(typeof(SharedMartialArtsSystem))]
+public sealed partial class MartialArtistComponent : Component
+{
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public ProtoId<MartialArtPrototype>? MartialArt;
+
+    [DataField]
+    public List<CombatSequenceStep> CurrentSteps = new();
+
+    [DataField]
+    public TimeSpan LastStepPerformedAt = TimeSpan.Zero;
+
+    [DataField]
+    public TimeSpan SequenceTimeout = TimeSpan.FromSeconds(2);
+
+    [DataField]
+    public float UpdateRate = 1; // per second
```

Comment:
```suggestion
    [DataField]
    public float UpdateFreqeuncy = 1; // in Hz
```
тикрейт 30 Hz апдейт у тебя 1 Hz, ща я посмотрю что ты там в апдейте делаешь, но какбудто апдейт тут не нужен..